### ISSUE TITLE
BC16.4 Error: The runtime version of the extension package is currently set to '5.4'. The runtime version must be set to '5.3' or earlier in the app.json file in order to install the extension package on this server.

### ISSUE THREAD

**Issue by DanielGoehler** [microsoft/AL#6687]
**1. Describe the bug**
AL Language Extensions says I need BC 16.4 for Runtime 5.4:
![image](https://user-images.githubusercontent.com/9556549/123934311-7453c900-d993-11eb-8ee0-a9310d2c5f9a.png)

But you can't publish this app in a BC 16.4 Environment. I get this error message:
`The request for path /dago-bc16-4dev/dev/apps?tenant=default&SchemaUpdateMode=synchronize&DependencyPublishingOption=default failed with code UnprocessableEntity. Reason: The runtime version of the extension package is currently set to '5.4'. The runtime version must be set to '5.3' or earlier in the app.json file in order to install the extension package on this server.
`
![image](https://user-images.githubusercontent.com/9556549/123934371-82a1e500-d993-11eb-825e-45adbb2a8636.png)

**Comment by thpeder** [comment]
Hi @DanielGoehler,
Thanks for reporting the issue.
It seems like when we worked on 16.4 and 16.5 then we didn't need to introduce a new runtime version for those release, but in the branch for 17.0 we had introduced the runtime versions 5.4 and 5.5.
Because you server uses a compiler without those versions, you can safely use runtime version 5.3, knowing that no new features were guarded behind those missing runtime versions.

**Comment by DanielGoehler** [comment]
Hi @thpeder, 
runtime version 5.3 doesn't support [LogMessage()](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/session/session-logmessage-string-string-verbosity-dataclassification-telemetryscope-dictionary[text,text]-method). So I have to use BC 17.0?

**Comment by thpeder** [comment]
yes if possible. I'm fixing it in 16, so it will be there in a future update.
